src/tests/system/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_executable(test_sleep test_sleep.f90)
 target_link_libraries(test_sleep fortran_stdlib)
 
-add_test(NAME Sleep COMMAND $<TARGET_FILE:test_sleep> 1234)
+add_test(NAME Sleep COMMAND $<TARGET_FILE:test_sleep> 650)
```

Comment:
Why not to reduce this to 10ms? 0.65s is still very much noticeable.

Reply:
If a mistake is made the test might just pass through without sleeping at all. Especially with the test run in parallel it shouldn't really add much time. so the test is set up to be a finite sleep time so that we can look back and see that yes indeed it actually was sleeping.

Reply:
I feel that's a very fragile way of testing sleeping, if it requires manual "eyeballing". And somebody can adjust these numbers in the future also. 

Isn't there some way of using [ctest_sleep](https://cmake.org/cmake/help/latest/command/ctest_sleep.html) to test this robustly with a shorter timeout?

Reply:
I think sleep() once it works as-is, would only be likely to break on systems where there are POSIX implementation issues. To enforce a minimum time in a platform agnostic way might be easiest done within Fortran by checking CPU_TIME. Trying to push these time durations too low can run into issues whether due to slow CPU or OS limitations like WIndows. Where the procedure setup/teardown can be longer than sleep()

Reply:
So how I have seen non-standard/proprietary sleep() fail is to hang forever (ifort) or not sleep at all (implementation mistakes)

Reply:
Ok. I am fine with the current solution for now.

Reply:
504f9b3 should address the auto-test of time.

Reply:
Thanks, I think that looks good.